Snake/BusinessLayer/AI.cs

```diff
@@ -11,40 +11,49 @@ public class AI
         private Direction currentDirection;
         private int X;
         private int Y;
-        Random randomNumber = new Random();
+        Random randomNumber;
         public int XCoordinate
         {
             get
             {
                 return X;
             }
+
+            set
+            {
+                X = value;
+            }
         }
 
+
         public int YCoordinate
         {
             get
             {
                 return Y;
             }
+            set
+            {
+                Y = value;
+            }
         }
 
-        public AI()
+        public AI(Random random)
         {
             X = -1;
             Y = -1;
+            randomNumber = random;
         }
 
-        public bool SpawnAI(int XBorder, int YBorder)
+        public void SpawnAI(int XBorder, int YBorder)
         {
             if (XBorder < 0 || YBorder < 0)
             {
-                return false;
+                throw new Exception("BAD");

```

Comment:
Update this exception - this was from my demo the other day - should be a better message!